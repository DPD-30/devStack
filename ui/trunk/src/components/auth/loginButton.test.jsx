import React from 'react'
import { HashRouter as Router } from 'react-router-dom'
import { renderToString } from 'react-dom/server'


import ComponentAuthLoginButton from './loginButton.jsx'

jest.unmock('./loginButton.jsx')


test('ComponentAuthLoginButton matches snapshot', () => {
	const comp = renderToString(<Router><ComponentAuthLoginButton /></Router>)

	expect(comp).toMatchSnapshot()
})
