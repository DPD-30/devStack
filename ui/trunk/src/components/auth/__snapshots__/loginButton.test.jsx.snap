// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentAuthLoginButton matches snapshot 1`] = `
<a href="#/login">
  SIGN IN
</a>
`;
